PowerEditor/src/WinControls/Preference/preference.rc

```diff
@@ -112,6 +112,9 @@ BEGIN
     CONTROL         "Enable smooth font",IDC_CHECK_SMOOTHFONT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,298,146,129,10
     CONTROL         "Enable scrolling beyond last line",IDC_CHECK_SCROLLBEYONDLASTLINE, "Button",BS_AUTOCHECKBOX | WS_TABSTOP,298,159,139,10
     CONTROL         "Disable advanced scrolling feature\r(if you have touchpad problem)",IDC_CHECK_DISABLEADVANCEDSCROLL, "Button",BS_AUTOCHECKBOX | BS_MULTILINE | WS_TABSTOP,298,172,139,18
+    GROUPBOX        "Comment Settings",IDC_COMMENT_SETTINGS_GB_STATIC,142,152,148,30,BS_CENTER
+    CONTROL         "Comment at start of Line",IDC_CHECK_LINE_COMMENT_AT_START_OF_LINE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,153,160,120,10
+    CONTROL         "Comment Blank Lines",IDC_CHECK_LINE_COMMENT_BLANK_LINE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,153,170,120,10

```

Comment:
Naming it IDC_CHECK_LINE_COMMENT_BLANK_LINE**S** would be more consistent with the description.